packages/xstate-inspect/src/browser.ts

```diff
@@ -1,14 +1,16 @@
 import {
   ActorRef,
-  Interpreter,
-  interpret,
-  EventObject,
   EventData,
+  EventObject,
+  interpret,
+  Interpreter,
   Observer,
-  toActorRef
+  toActorRef,
+  toEventObject,
+  toObserver,
+  toSCXMLEvent,
+  XStateDevInterface
 } from 'xstate';
-import { XStateDevInterface } from 'xstate/lib/devTools';
-import { toSCXMLEvent, toEventObject, toObserver } from 'xstate/lib/utils';
```

Comment:
Such changes should be OK as per our semver policy: https://github.com/statelyai/xstate/tree/50c271dc1a1a05b035364f8247aa4d80d613864f#packages-1